packages/core/src/css/LibCssExtractLoader.ts

```diff
@@ -0,0 +1,258 @@
+/**
+ * The following code is modified based on
```

Comment:
write some comments here to implement what you modified

Reply:
Do we need to sync this file in the future?

Reply:
Usually not, cssExtract loader will only need to be adjusted after the css-loader is changed, but it may be a very difficult breaking change to advance.